corehq/apps/export/tests/test_export_conversion.py

```diff
@@ -72,6 +72,30 @@ def test_toggle_turned_on(self, _):
         self.assertTrue(toggle_enabled(NEW_EXPORTS.slug, self.domain, namespace=NAMESPACE_DOMAIN))
 
 
+class TestIsFormStockExportQuestion(SimpleTestCase):
+    """Ensures that we can guess that a column is a stock question"""
+
+
+@generate_cases([
+    ('form.balance.entry.@id', True),
+    ('form.balance.entry.@notme', False),
+    ('form.balance.@date', True),
+    ('form.balance.@notme', False),
+
+    ('form.transfer.entry.@id', True),
+    ('form.transfer.entry.@notme', False),
+    ('form.transfer.@date', True),
+    ('form.transfer.@notme', False),
+
+    ('form.notme.@quantity', False),
+    ('tooshort', False),
+    ('tooshort.tooshort', False),
+    ('', False),
+], TestIsFormStockExportQuestion)
+def test_is_form_stock_question(self, index, expected):
```

Comment:
Is this function at the wrong indent level? Looks like it is at the same level as `TestIsFormStockExportQuestion`